app/assets/src/components/views/TaxonTreeVis.jsx

```diff
@@ -30,15 +31,33 @@ class TaxonTreeVis extends React.Component {
 
     this.metrics = {
       aggregatescore: {
-        label: "Aggregate Score",
+        label: METRIC_NAMES["aggregatescore"],
```

Comment:
No need to add label here. Remove the labels here and call `METRIC_NAMES[metric]` below instead.

Reply:
thanks